docs/sources/write/image-guidelines/index.md

```diff
@@ -193,14 +194,18 @@ General rules:
 
 ### Recommended image editors
 
+<!-- vale Grafana.Spelling = NO -->
+
 - Linux: Use [gimp](https://www.gimp.org/)
 - macOS: Use [Snagit](https://www.techsmith.com/screen-capture.html)
 - Windows: Use [Snagit](https://www.techsmith.com/screen-capture.html)
 - Web browser: Use [Photopea](https://www.photopea.com/)
 
+<!-- vale Grafana.Spelling = YES -->
+
 ## Where to store media assets
 
-All visual assets are stored in Google Cloud Storage, only accessible to Grafana Labs employees. You use the [asset uploader application](https://admin.grafana.com/upload/) to upload assets.
+All visual assets are stored in Google Cloud Storage, only accessible to Grafana Labs employees. You use the [asset upload application](https://admin.grafana.com/upload/) to upload assets.
 The following table lists the steps you take to provide the Grafana Labs technical documentation team with the image.
```

Comment:
```suggestion
The following table lists the steps you take to provide us with the image.
```

Reply:
I'm not sure if a reader will have enough context to understand who "us" is.

That being said, I'm not sure how the reader knows who "the Grafana Labs technical documentation team" is either.